eng/scripts/Test-Packages.ps1

```diff
@@ -59,33 +54,20 @@ foreach ($package in $packagesToTest) {
 
     Write-Host "`n`nTesting package: '$($package.Name)'`n"
 
-    Invoke-LoggedCommand "cargo +$Toolchain build --keep-going"
+    Invoke-LoggedCommand "cargo +$Toolchain build --keep-going" -GroupOutput
     Write-Host "`n`n"
 
-    $targets = @()
-    if ($UnitTests) {
-      $targets += "--lib"
-    }
-
-    if ($FunctionalTests) {
-      $targets += "--bins"
-      $targets += "--examples"
-      $targets += "--tests"
-      $targets += "--benches"
-    }
-
-    Invoke-LoggedCommand "cargo +$Toolchain test $($targets -join ' ') --no-fail-fast"
+    Invoke-LoggedCommand "cargo +$Toolchain test --doc --no-fail-fast" -GroupOutput
     Write-Host "`n`n"
 
-    Invoke-LoggedCommand "cargo +$Toolchain test --doc --no-fail-fast"
+    Invoke-LoggedCommand "cargo +$Toolchain test --all-targets --no-fail-fast" -GroupOutput
```

Comment:
We should add `--all-features` as well.

/cc @analogrelay

Reply:
should we add `--all-features` to both calls?

Reply:
added